notification/notification-sdk/src/main/java/org/sunbird/notification/email/service/impl/SmtpEMailServiceImpl.java

```diff
@@ -27,18 +27,12 @@ public boolean sendEmail(EmailRequest emailReq) {
     if (emailReq == null) {
       logger.info("Email request is null or empty:");
       return false;
-      // either email object has bcc or to list size more than 1 then pass it as bcc.
-    } else if (CollectionUtils.isNotEmpty(emailReq.getBcc()) || emailReq.getTo().size() > 1) {
-      return email.sendEmail(
-          email.getFromEmail(),
-          emailReq.getSubject(),
-          emailReq.getBody(),
-          CollectionUtils.isEmpty(emailReq.getBcc()) ? emailReq.getTo() : emailReq.getBcc());
-    } else if (CollectionUtils.isNotEmpty(emailReq.getCc())) {
-      return email.sendMail(
-          emailReq.getTo(), emailReq.getSubject(), emailReq.getBody(), emailReq.getCc());
-    } else {
-      return email.sendMail(emailReq.getTo(), emailReq.getSubject(), emailReq.getBody());
+    }
+    else {
+      logger.info("Email Request To List : " + emailReq.getTo());
```

Comment:
As mentioned in the other PR - remove the info logger

Reply:
Removed in the previous PR, kindly verify